### PR TITLE
add opencv support to Dockerfile

### PR DESCRIPTION
**Pull request by keineahnung2345**
cv2 is used by predictor.py, so it's required

### CODE REVIEW

**Comment by miguelvr** [comment]
I would prefer just to keep it in conda, as it reduces the boilerplate and guarantees a correct installation

File: docker/Dockerfile [hunk 1]
```diff
@@ -28,7 +30,7 @@ ENV PATH=$CONDA_PREFIX/bin:$PATH
 ENV CONDA_AUTO_UPDATE_CONDA=false
 
 RUN conda install -y ipython
-RUN pip install ninja yacs cython matplotlib
+RUN pip install ninja yacs cython matplotlib opencv-python
 
 # Install PyTorch 1.0 Nightly
 RUN conda install -y pytorch-nightly -c pytorch && conda clean -ya
```
Comment (miguelvr) [inline docker/Dockerfile]: Replace with

```
# Install PyTorch 1.0 Nightly and OpenCV
RUN conda install -y pytorch-nightly -c pytorch \
 && conda install -y opencv -c menpo \
 && conda clean -ya
```

and remove your added apt-get install (...)

Comment (keineahnung2345) [reply docker/Dockerfile]: I've tried testing the effect of installing opencv through conda, and it works without my apt-get install xxx, thanks!

**Comment by keineahnung2345** [comment]
@miguelvr I have changed to conda. But I wonder why you install ninja, yacs, cython, matplotlib through pip?

**Comment by fmassa** [comment]
@keineahnung2345 that's the default requirements for the installation for those libs, and is what we recommend in the installation.

**Comment by miguelvr** [comment]
@keineahnung2345 basically I just put it as in the INSTALL.md for clarity.

The benefit of installing opencv through conda is just to avoid the clutter of all the dependencies it has

**Pull request merged**